tests/config_test.py

```diff
@@ -102,6 +102,25 @@ def test_get_config_values(self):
         values = self.namespace.get_config_values()
         assert_equal(values, {'stars': 'foo'})
 
+    def test_get_config_dict(self):
+        self.namespace['one.two.three.four'] = 5
+        self.namespace['a.b'] = 'c'
+        self.namespace['first'] = True
```

Comment:
It'd be good to have some overlap here so that the test verifies that overlapping subkeys don't clobber each other.